src/deluge/gui/views/automation_view.cpp

```diff
@@ -3294,6 +3326,12 @@ void AutomationView::setParameterAutomationValue(ModelStackWithAutoParam* modelS
 // deleting automation, or displaying current parameter value
 // multi pad presses don't use this function
 void AutomationView::setKnobIndicatorLevels(int32_t knobPos) {
+	// if you're dealing with a patch cable which has a -128 to +128 range
+	// we'll need to convert it to a 0 - 128 range for purpose of rendering on knob indicators
+	if (!onArrangerView && (getCurrentClip()->lastSelectedParamKind == params::Kind::PATCH_CABLE)) {
+		float floatKnobPos = kMaxKnobPos * ((static_cast<float>(knobPos) + kMaxKnobPos) / (kMaxKnobPos * 2));
```

Comment:
Ideally they'd show bipolar, maybe negative values fill in from the top. 

Currently this doesn't match the display of learning gold knobs to patch cables either (which is positive only). I think they need to match either way - so only positives in automation right now, or fix both

Reply:
I will be fixing both so that the gold knobs outside automation view are bipolar as well. Thought it would be best to do that in a separate PR. It should be bipolar so that the middle of the LED indicators is 0, just like with pan.